addons/medical/functions/fnc_canTreat.sqf

```diff
@@ -16,17 +16,16 @@
 
 #include "script_component.hpp"
 
-private ["_caller", "_target", "_selectionName", "_className", "_config", "_medicRequired", "_items", "_locations", "_return", "_condition", "_patientStateCondition", "_allowedSelections"];
-_caller = _this select 0;
-_target = _this select 1;
-_selectionName = _this select 2;
-_className = _this select 3;
+private ["_config", "_medicRequired", "_items", "_locations", "_return", "_condition", "_patientStateCondition", "_allowedSelections"];
+params ["_caller", "_target", "_selectionName", "_className"];
 
-if !(_target isKindOf "CAManBase") exitWith {false};
+if !(_target isKindOf "CAManBase") exitWith { false };
 
-_config = (ConfigFile >> "ACE_Medical_Actions" >> "Basic" >> _className);
-if (GVAR(level)>=2) then {
-    _config = (ConfigFile >> "ACE_Medical_Actions" >> "Advanced" >> _className);
+
+_config = if (GVAR(level)>=2) then {
+    (ConfigFile >> "ACE_Medical_Actions" >> "Advanced" >> _className)
+} else {
+    (ConfigFile >> "ACE_Medical_Actions" >> "Basic" >> _className)
```

Comment:
posible use of this _config = (ConfigFile >> "ACE_Medical_Actions" >> (["Basic", "Advanced"] select (GVAR(level)>=2)) >> _className)